extra/2-piping.js

```diff
@@ -16,27 +16,28 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  return a + b;
 }
 
-function multiply() {
-
+function multiply(a, b) {
+  return a * b;
 }
 
-function format() {
-
+function format(number) {
+  return `£${number.toString()}`;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = format(multiply(add(startingValue, 10), 2));
 
 /* BETTER PRACTICE */
-
-let goodCode =
-
+let sum = add(startingValue, 10);
+let product = multiply(sum, 2);
+let formatted = `£${product}`;
```

Comment:
is this going to always produce the correct two decimal places?

Reply:
No, but I have added Math.round() to product because the tests provided doesn't want a decimal value.